koji_containerbuild/plugins/cli_containerbuild.py

```diff
@@ -105,6 +105,9 @@ def parse_arguments(options, args, flatpak):
                       help=_("ODCS composes used. May be used multiple times. Cannot be"
                              " used with --signing-intent"),
                       dest='compose_ids', action='append', metavar="COMPOSE_ID", type="int")
+    parser.add_option("--skip-build", action="store_true",
+                      help=_("Don't create build, but just modify settings for autorebuilds"))
```

Comment:
I'd omit part about autorebuilds here. It can be used for any settings right?

Reply:
I mean yes it can be used for updating any buildconfig, but unless it is autorebuild, updating buildconfig without build is basically meaningless

Reply:
This should be meaningful for the users, right?

Could we word it like "Skip build and update buildconfig. Use this option to update autorebuild settings"

Reply:
sure sounds good will change

Reply:
changed